publication-event-handlers/src/test/java/no/unit/nva/publication/events/handlers/tickets/DoiRequestEventProducerTest.java

```diff
@@ -278,7 +278,7 @@ private URI extractExpectedCustomerId(Publication publication) {
     }
 
     private URI inferExpectedPublicationId(Publication publication) {
-        return UriWrapper.fromUri(NVA_API_DOMAIN)
+        return UriWrapper.fromHost(new Environment().readEnv("API_HOST"))
```

Comment:
This test was exact the same as in production code, so we did not test that expected value is actually the expected value.